integrations/Matomo/util.js

```diff
@@ -57,14 +57,13 @@ const standardEventsMapping = (event, standardEventsMap, message) => {
 
           case "trackSiteSearch":
             if (properties) {
-              keyword = properties.keyword || properties.search;
               category = properties.category;
               resultsCount = properties.resultsCount;
             }
-            if (!keyword || keyword === "")
-              keyword = message.context
-                ? message.context.page.search
-                : undefined;
+            keyword =
+              properties?.keyword ||
```

Comment:
properties will be already present no need of optional chaining for it

Reply:
@ItsSudip your thoughts

Reply:
@utsabc do you mean in this way?
`keyword= properties.keyword || properties.search || message.context.page?.search;`

Reply:
Done

Reply:
yes